src/k8s-engine/lima.ts

```diff
@@ -400,8 +401,13 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   protected async lima(...args: string[]): Promise<void> {
     const stream = await Logging.lima.fdStream;
 
-    await childProcess.spawnFile(this.limactl, args,
-      { env: this.limaEnv, stdio: ['ignore', stream, stream] });
+    try {
+      await childProcess.spawnFile(this.limactl, args,
+        { env: this.limaEnv, stdio: ['ignore', stream, stream] });
+    } catch (ex) {
+      console.log(`+ limactl ${ args.join(' ') }`);
```

Comment:
We immediately throw again, which will (eventually) dump the error.

But yes, printing the error here would make sense too, in case other things got intermingled.